values.yaml

```diff
@@ -3,6 +3,8 @@ images:
   bash: docker.io/library/bash:latest
   skopeo: quay.io/skopeo/stable:latest
   buildah: registry.access.redhat.com/ubi8/buildah:latest
+  s2igo: quay.io/openshift-pipeline/s2i:latest
+  s2ibuildah: quay.io/buildah/stable:latest
```

Comment:
It should be using the same `buildah` image referenced above.

Reply:
Actually here the s2igo and s2ibuildah images are redundant. I mentioned them here keeping in mind that in future if I separate the commands used in the s2i go task as separate scripts. Should I remove them?